src/config/dynamic-theme-fixes.config

```diff
@@ -291,6 +291,20 @@ INVERT
 
 ================================
 
+br.de
+
+INVERT
+article h4
+article h3
+article .css-lip0i6
+
+CSS
+article section, article footer, .css-1fzo8jw {
+    background-color: rgba(0, 0, 0, 0.75) !important;
```

Comment:
`background-color: ${white} !important;` should be used instead to make the fix work in Light mode. Instead of opacity it is better to define a solid color shade, because for large areas it can have performance problems.